logic app templates/latency-events-template.json

```diff
@@ -0,0 +1,241 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Logic App Name": {
+      "defaultValue": "parse-dfp-audit-events-template",
```

Comment:
I think this default value should change to not collide with the audit events one.